src/constants/defaultSettingsValues.js

```diff
@@ -0,0 +1,12 @@
+const DEFAULT_PAGE_SIZE = 10;
+const defaultSettingsValues = {
+  pageSize: {
+    agreementLines: DEFAULT_PAGE_SIZE,
+    agreementEresources: DEFAULT_PAGE_SIZE,
+    entitlementOptions: DEFAULT_PAGE_SIZE,
+    packageContents: DEFAULT_PAGE_SIZE,
+    entitlements: DEFAULT_PAGE_SIZE
+  },
+};
+
+export default defaultSettingsValues;
```

Comment:
`defaultSettingsValues` is too generic to tell what this file exports. Maybe change it to `defaultMclsPageSize.js` or similar